py-scripts/lf_interop_qos.py

```diff
@@ -1495,22 +1662,29 @@ def main():
         throughput_qos.build()
         throughput_qos.start(False, False)
         time.sleep(10)
-        connections_download, connections_upload, drop_a_per, drop_b_per = throughput_qos.monitor()
+        connections_download, connections_upload, drop_a_per, drop_b_per, connections_download_avg, connections_upload_avg, avg_drop_a, avg_drop_b = throughput_qos.monitor()
```

Comment:
Again bad style returning this many parameters (nothing to change for now). In the future, maybe a test object w/ relevant data would be better